crates/solvers/src/infra/config/dex/file.rs

```diff
@@ -63,6 +71,8 @@ pub async fn load<T: DeserializeOwned>(path: &Path) -> (super::Config, T) {
             config.absolute_slippage.map(eth::Ether),
         )
         .expect("invalid slippage limits"),
+        concurrent_requests: NonZeroUsize::new(config.concurrent_requests)
+            .expect("invalid concurrent requests values"),
```

Comment:
Would be more helpful to state that that value has to be greater than 0.

Reply:
Serialized directly to the non-zero type.